spark/dl/pom.xml

```diff
@@ -252,17 +251,6 @@
                     </execution>
                 </executions>
             </plugin>
-
-            <plugin>
-                <groupId>net.alchim31.maven</groupId>
-                <artifactId>scala-maven-plugin</artifactId>
```

Comment:
The scala-maven-plugin here is introduced to exclude the "structured streaming" files (see pr #876 ). There is another scala-maven-plugin configuration in "bigdl-parent.pom"